MSAL/src/MSALPublicClientApplication.m

```diff
@@ -133,6 +138,9 @@ - (id)initWithClientId:(NSString *)clientId
     MSIDDefaultTokenCacheAccessor *defaultAccessor = [[MSIDDefaultTokenCacheAccessor alloc] initWithDataSource:dataSource otherCacheAccessors:@[legacyAccessor] factory:factory];
     
     self.tokenCache = defaultAccessor;
+    
+    _webviewSelection = MSALWebviewSelectionDefault;
```

Comment:
we only set default for TARGET_OS_IPHONE?

Reply:
Nope!